docs/faq.md

```diff
@@ -1,24 +1,22 @@
 # Frequently Asked Questions
 
-<!--
-The TOC for this document was generated using https://github.com/ekalinin/github-markdown-toc.go.
-
-# Install gh-md-toc
-brew install github-markdown-toc
-
-# Generate TOC
-gh-md-toc --hide-header --hide-footer --start-depth=1
--->
-* [Why are there multiple versions of some of my targets?](#why-are-there-multiple-versions-of-some-of-my-targets)
-* [My Xcode project seems to be of of sync with my Bazel project\. What should I do?](#my-xcode-project-seems-to-be-of-of-sync-with-my-bazel-project-what-should-i-do)
-* [When I build I get warnings like "Stale file 'PROJECT\.xcodeproj/rules\_xcodeproj/gen\_dir/\.\.\.' is located outside of the allowed root paths"\. How do I fix this?](#when-i-build-i-get-warnings-like-stale-file-projectxcodeprojrules_xcodeprojgen_dir-is-located-outside-of-the-allowed-root-paths-how-do-i-fix-this)
-* [Why do I get an error like "Provisioning profile "PROFILE\_NAME" is Xcode managed, but signing settings require a manually managed profile\. (in target 'TARGET' from project 'PROJECT')"?](#why-do-i-get-an-error-like-provisioning-profile-profile_name-is-xcode-managed-but-signing-settings-require-a-manually-managed-profile-in-target-target-from-project-project)
-* [Why do I get an error like "No profile for team 'TEAM' matching 'PROFILE\_NAME' found: Xcode couldn't find any provisioning profiles matching 'TEAM\_ID/PROFILE\_NAME'\. Install the profile (by dragging and dropping it onto Xcode's dock item) or select a different one in the Signing &amp; Capabilities tab of the target editor\."?](#why-do-i-get-an-error-like-no-profile-for-team-team-matching-profile_name-found-xcode-couldnt-find-any-provisioning-profiles-matching-team_idprofile_name-install-the-profile-by-dragging-and-dropping-it-onto-xcodes-dock-item-or-select-a-different-one-in-the-signing--capabilities-tab-of-the-target-editor)
-* [What is CompileStub\.m?](#what-is-compilestubm)
-* [Why do some of my `swift_library`s compile twice in BwX mode?](#why-do-some-of-my-swift_librarys-compile-twice-in-bwx-mode)
-* [Do I need to place my custom Xcode scheme declarations in a function like `tools/generator`?](#do-i-need-to-place-my-custom-xcode-scheme-declarations-in-a-function-like-toolsgenerator)
-* [Why does `tools/generator` declare its custom Xcode schemes in a-function?](#why-does-toolsgenerator-declare-its-custom-xcode-schemes-in-a-function)
-* [Does the Archive action work?](#does-the-archive-action-work)
+## What do BwB and BwX mean?
+
+The `xcodeproj` rule supports two different
+[build modes](bazel.md#xcodeproj-build_mode), `"bazel"` and `"xcode"`, commonly
+referred to as Build with Bazel (BwB) and Build with Xcode (BwX).
+
+## Does the Archive action work?
+
+Currently no effort is put into making the Archive action work correctly.
```

Comment:
I wonder if we should prevent the archive action from being enabled in all schemes to make this more clear?